pkg/filters/filter_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+package filters
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"storj.io/storj/pkg/storj"
+)
+
+var pieceIDs [][]byte
+var initDone bool
+var nbPiecesInFilter int
+var totalNbPieces int
+var falsePositiveProbability float64
+
+//  generates 1 million piece ids
+// adds 95% of them to the bloom filter,
+// and then checks all 1 million piece ids with the bloom filter
+// measure times, memory allocation, false positives
+
+func Init() {
```

Comment:
There is no need to be capitalized.

However we can achieve what `Init` does using a [`TestMain`](https://golang.org/pkg/testing/#hdr-Main) function and avoid any potential data race and also influence in the first bench march function which runs and call `Init`.

```go
func TestMain(m *testing.M) {                                                                                              
  totalNbPieces = 1000000                                                                                                  
  nbPiecesInFilter = 950000                                                                                                
  pieceIDs = GenerateIDs(totalNbPieces)                                                                                    
  initDone = true                                                                                                          
  falsePositiveProbability = 0.1                                                                                           
  os.Exit(m.Run())                                                                                                         
} 
```